proto/src/determined/api/v1/master.proto

```diff
@@ -22,6 +22,9 @@ message SSOProvider {
   string sso_url = 2;
   // The type of SSO (such as SAML, OIDC).
   string type = 3;
+  // The flag to indicates if this provider should be always be redirected to,
```

Comment:
super nit picky: I think there's a typo here `The flag to indicates`

Reply:
All good this is a solid catch! Thanks!